### ISSUE TITLE
validation loss get saturated 

### ISSUE THREAD

**Issue by amindehnavi** [qfgaohao/pytorch-ssd#160]
Hi everybody 
I use VOC dataset to train MobileNet_v2. first I separate the VOC dataset and use the images that contain only person labels. then I start training the MobileNet_v2 with these images and their corresponding labels, but after about 100 epochs, validation loss get saturated and doesn't decrease (I'm sure that my dataset is correct!). 

To make sure the network is working properly, use different optimizer like Adam and learning rate schedulers like MULTISTEPLR. ONECYCLELR, REDUCELRONPLATEAU and COSINEANNEALINGLR all from Torch implemented schedulers. also I tested different scenarios in freezing backbone and extra layers. unfortunately in the best case the accuracy of trained net reached to %68 for single class detection i.e. Person. while in repository is said that they have reached to %68 on 20 class dataset and I expected to reach about %80 in accuracy for person detection (furthermore I use pretrained weights that exist in this repository).  

does anyone has an idea what should I do to increase the accuracy? 

Thanks
